ClientApp/src/components/shared/navbar/NavBarPublic.tsx

```diff
@@ -77,7 +77,7 @@ const NavBarPublic = () => {
                   </Scroll>
                 </ListItem>
                 <ListItem>
-                  <Scroll onClick={handleClose} to="questions" smooth={true}>
+                  <Scroll onClick={() => {history.push("/spørsmål");}} to="questions" smooth={true}>
```

Comment:
All or most of the <Scroll> components should be replaced, not sure if we do that in the scope of this PR or create a new issue

Reply:
I think <Link> might work